src/api.js

```diff
@@ -608,6 +608,8 @@ api.declare({
 }, async function(req, res) {
   var taskId  = req.params.taskId;
   var taskDef = req.body;
+  var clientId_default = await
```

Comment:
We don't want to mix capitalization styles.  This would be spelled `clientIdDefault` instead.  But, it's not just the default, it's the only option, so perhaps just `clientId` or `callerClientId`?

Reply:
ok will modify